lib/Kconfig.kfence

```diff
@@ -22,3 +22,15 @@ config KFENCE
 	  such as KASAN. KFENCE can detect a subset of bugs that KASAN can
 	  detect (therefore enabling KFENCE together with KASAN does not make
 	  sense), albeit at very different performance profiles.
+
+if KFENCE
+
+config KFENCE_SAMPLE_RATE
+	int "KFENCE default sample rate in milliseconds"
+	default 100
```

Comment:
Sounds good. Android ppl may want to set the rate in .config rather than boot parameters.